app/serializers/rest/admin/existing_domain_block_error_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class REST::Admin::ExistingDomainBlockErrorSerializer < ActiveModel::Serializer
```

Comment:
This is a completely new API design, we have never used custom error serializers before. However, I believe there might be a more generalized validation errors rendering system that I added for the sign-up API, perhaps it could be reused?

Reply:
I am not sure how useful it would be. Here, we're specifically interested in knowing *which* domain block it conflicts with so that the client app can show it on conflict. I am not completely sure how that would fit.